index.html

```diff
@@ -228,37 +250,38 @@ <h2 class="aboutmehead">About me</h2>
         </ul>
     </div>
 </div>
-
-</article>
 </section>
+
 <footer class="footer">
     <div class="containers">
-    <div  class="textheads" >
-    <h2 class="contactheads">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
-    <form action="https://formspree.io/f/xrgdkqrd" method="post" class="myforms">
-        <ul class="textfields">
-            <li>
-                <hr class="liners">
-        <input type="text" placeholder="Full name" id="name" name="name" maxlength="30" required>
-        </li>
-        
-        <li>
-            <hr class="liners2">
-            <input type="email" placeholder="Email address" id="email" name="email-address" required >
-        </li>
-        <li>
-            <button type="submit" class="button button2">Get in touch</button>
-        </li>
-        <li>
-            <div class="textcontainer">
-            <textarea placeholder="Enter text here" id="textarea" name="comment" maxlength="500" required></textarea>
+        <div class="textheads">
+            <h2 class="contactheads">I'm always interested in hearing about new projects, so if you'd like to chat
+                please get in touch.</h2>
         </div>
-        </li>
-        </ul>
+        <form action="https://formspree.io/f/xrgdkqrd" method="post" class="myforms">
```

Comment:
Kindly Fix your button alignment here. To make your form more beautiful.

![image](https://user-images.githubusercontent.com/34779928/194695892-a200757b-ea21-4b63-9ee6-2752a2742b0c.png)